src/main/java/invtweaks/InvTweaksHandlerShortcuts.java

```diff
@@ -437,16 +437,18 @@ private void moveAll(ShortcutConfig shortcut, ItemStack stackToMatch) {
 
                     // Continue if movement succeeded, there is another slot to try, or we're dropping items.
                     // In reverse: fail if movement failed, AND there are no other slots AND we're not dropping.
-                    if(success || (newIndex != toIndex) || (shortcut.action == ShortcutSpecification.Action.DROP)) {
+                    if (newIndex == toIndex) {
```

Comment:
actually it happened..., sometime I am sorting or dropping some item. this will turn into infinite loop.....
as I remember, the success will be true, and newIndex and toIndex will always same.